### DISCUSSION TITLE
cryptosmite source

### DISCUSSION THREAD

**Discussion by slimepointe** [3kh0/ext-remover#953] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
yeah repo on my profile for cryptosmite source
u cant use it tho but ill drop the statefuls soon

**Comment by CGRmags** [comment]
@slimepointe interesting can you re-enroll after doing this?

**Reply by CGRmags** [reply]
@AshtonDavies Would this be better or is Skiovox better for playing games.

**Reply by AshtonDavies** [reply]
You should be able to re-enroll with Sh1mmer I believe, but we don't really know quite how Cryptosmite will work yet, so there's probably a small chance that the operation wouldn't work the same on later versions while unenrolled a different way, considering that they had to get around the patch to do so. That may not even matter though, but I am not a part of the Mercury Workshop team, so I don't know the specifics of how that would work.

As you know, unenrollment gives you more privileges on what games you can play. If you're just into browser games, there's no need to remove enterprise enrollment just to do that. But if you're a civilized being who prefers the real deal such as Minecraft, some Steam, and other games, you should pick Cryptosmite. Sorry if that was obvious to you already. It's better to stay enrolled but caubed if you just want to browse without filters.

**Reply by CGRmags** [reply]
@AshtonDavies thanks lol

**Reply by CGRmags** [reply]
@AshtonDavies i am civilized trust

**Comment by v4pl** [comment]
Cryptosmite out now?

**Reply by v4pl** [reply]
@AshtonDavies

**Reply by AshtonDavies** [reply]
I don't think so.

**Reply by v4pl** [reply]
then what is this about?

**Reply by slimepointe** [reply]
look at my profile, cryptosmite source code.

**Reply by MunyDev** [reply]
ok
there is no need to use his version of cryptosmite, we have a better and more official version taht will release today

**Comment by ProgrammerIn-wonderland** [comment]
This is so sad, by the time you figure out how to make strateful it will probably be officially released LMAO
